misc_docs/syntax/operator_ref_assignment.mdx

```diff
@@ -0,0 +1,26 @@
+---
+id: "ref-assignment"
+keywords: ["ref", "assign"]
+name: ":="
+summary: "This is the `ref assignment` operator."
+category: "operators"
+---
+
+This operator is syntactic sugar for assigning to the `contents` value of a `ref`
```

Comment:
Let's keep it simple with the wording, no need to talk about the technical details if this is syntax sugar / a function. To be exact, `:=` is a whitelisted operator within the syntax, so it's kinda both, a function defined in `Pervasives`, and "special sugar"  in the parser.

```suggestion
This operator assigns a new value to a `ref`.
```